digid_eherkenning/digid_urls.py

```diff
@@ -7,4 +7,6 @@
 urlpatterns = [
     path("login/", views.DigiDLoginView.as_view(), name="login"),
     path("acs/", views.DigiDAssertionConsumerServiceView.as_view(), name="acs"),
+    path("logout/", views.DigiDLogoutView.as_view(), name="logout"),
+    path("slo/", views.DigidSingleLogoutView.as_view(), name="slo"),
```

Comment:
I'm thinking that it might be good to call this view `DigiDLogoutCallbackView` or `DigiDSingleLogoutCallbackView` so that it makes it easier in the future to remember what the difference is with `DigiDLogoutView`?